### PR TITLE
 modified grammar in the CHANGELOG.md file

### PR DESCRIPTION
**Pull request by Mr-DG-Wick**
small grammatical changes to the CHANGELOG.md file. Added some missing words and rephrased some paragraphs for better readability.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -155,7 +155,7 @@ This release is brought to you by [Sasha Koss](http://github.com/kossnocorp), [M
 
 ## v2.19.0 - 2021-03-05
 
-On this release worked [Tan75](http://github.com/tan75).
+[Tan75](http://github.com/tan75) wroked on this release.
```
Comment (tan75) [inline CHANGELOG.md:158]:
```suggestion
[Tan75](http://github.com/tan75) worked on this release.
```

**Review by tan75 — APPROVED** [review]
Thank you! We will accept this PR however we will not be able to change the old changelogs.

File: CHANGELOG.md (hunk 1 above)
Comment (Mr-DG-Wick) [reply CHANGELOG.md]: thanks for the reply tan!.  I am looking forward to contribute more :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -560,23 +560,23 @@ If you're upgrading from v2 alpha or beta, [see the pre-release changelog](https
...
+- Fixed a few bugs that appeared in timezones with offsets that include seconds (e.g. GMT+00:57:44).
   See PR [#789](https://github.com/date-fns/date-fns/pull/789).
 
 - [Fixed DST issue](https://github.com/date-fns/date-fns/pull/1003). See [#972](https://github.com/date-fns/date-fns/issues/972) and [#992](https://github.com/date-fns/date-fns/issues/992) for more details.
 
 - Fixed DST issue in `eachDayOfInterval` that caused time in the days
   after DST change to have the shift as well.
 
-- Fix bug in Galician locale caused by incorrect usage of `getHours`
+- Fixed` bug in Galician locale caused by incorrect usage of `getHours`
```
Comment (kossnocorp) [inline CHANGELOG.md:579]:
```suggestion
- Fixed bug in Galician locale caused by incorrect usage of `getHours`
```

**Review by kossnocorp — DISMISSED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (Mr-DG-Wick) [reply CHANGELOG.md]: thanks for the feedback!. 
Committed new changes according to your suggestion.

**Review by tan75 — APPROVED** [review]
🙏

**Pull request merged**